pkg/actors/actors.go

```diff
@@ -1226,7 +1233,7 @@ func (a *actorsRuntime) getActorTypeMetadata(actorType string, migrate bool) (*A
 
 	if !a.actorTypeMetadataEnabled {
 		return &ActorMetadata{
-			ID: uuid.NewString(),
+			ID: metadataZeroID,
```

Comment:
This is the default metadata ID when porting the "zero partition" configuration. We should keep the ID the same for idempotency.